### DISCUSSION TITLE
VS Code Gremlins extension can show hidden stuff

### DISCUSSION THREAD

**Discussion by peterbe** [mdn/mdn-community#495] [Content]
If you use VS Code to work on content, I can suggest [Gremlins tracker](https://marketplace.visualstudio.com/items?itemName=nhoizey.gremlins).
It's quite nice to see what all those hidden zero-width space Unicode characters are.

Example
<img width="848" alt="Screen Shot 2021-06-03 at 12 12 41 PM" src="https://user-images.githubusercontent.com/26739/120677481-1cde3e00-c465-11eb-930b-4f58d73626ba.png">

**Comment by pepelsbey** [comment]
Thanks for the tip! Closing it since it didn’t get much traction since.
